script.skinshortcuts/addon.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.skinshortcuts" name="Skin Shortcuts" version="1.1.0" provider-name="BigNoid, marcelveldt, SiLVO">
+	<requires>
+		<import addon="xbmc.python" version="3.0.0"/>
+		<import addon="script.module.simplejson" version="3.3.0"/>
```

Comment:
No need for script.module.simplejson. It's not used anywhere in the code and you already switched to the included "json" module.